modules/op-release-notes-1-10.adoc

```diff
@@ -152,4 +152,17 @@ $ oc get tektoninstallersets.operator.tekton.dev | awk '/pipeline-main-static/ {
 * This update adds a conversion configuration to the `ResolutionRequest` CRD. This update properly configures conversion from the `v1alpha1.ResolutionRequest` request to the `v1beta1.ResolutionRequest` request.
 * This update checks for duplicate workspaces associated with a pipeline task.
 * This update fixes the default value for enabling resolvers in the code.
-* This update fixes `TaskRef` and `PipelineRef` names conversion by using a resolver.
\ No newline at end of file
+* This update fixes `TaskRef` and `PipelineRef` names conversion by using a resolver.
+
+[id="release-notes-1-10-1_{context}"]
+== Release notes for {pipelines-title} General Availability 1.10.1
+
+With this update, {pipelines-title} General Availability (GA) 1.10.1 is available on {product-title} 4.11 and later versions.
+
+[id="fixed-issues-1-10-1_{context}"]
+=== Fixed issues for {pac}
+
+* Before this update, if the source branch information coming from payload included `refs/heads/` but the user-configured target branch only included the branch name, `main`, in a CEL expression, the push request would fail. With this update, {pac} passes the push request and triggers a pipeline if either the base branch or target branch has `refs/heads/` in the payload.
```

Comment:
Is `main` the only branch name where the error happened or just a sample branch name? If the latter, I would put "such as" before it.

Reply:
Yes, the `main` branch is the only branch where this error occured when you would define a CEL expresion for `event=push` and `target_branch=main`.